source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,68 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``Collection.UpdateOne()``
+method. 
+
+This example passes a query filter and an update parameter to the
+``UpdateOne()`` method. The query filter specifies the document to
+match, while the update parameter specifies the change to
+make to the document. The following code example calls the
+``UpdateOne()`` method on a document matched by its ``_id`` field from
+the ``restaurants`` collection to create a new field called
+``avg_rating`` with a value of 4.4:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateOne.go
+   :start-after: begin updateone
+   :end-before: end updateone
+   :emphasize-lines: 6
+   :language: go
+   :dedent:
+
+Click here <TODO: github link to file> to see a fully runnable example.
+
+.. note::
+
+   If the query filter does not match any documents, the operation will
+   return a result with a ``MatchedCount`` property of 0. If the filter
+   matches multiple documents, the method will select and update one document.
+
+Expected Result
+---------------
+
+If you execute a :doc:`find
+</fundamentals/crud/read-operations/retrieve/#find>` operation querying
+for the same ``_id`` value used in the example, you should see that the
+matched document now includes the ``avg_rating`` field:
```

Comment:
nit: Let's clarify this language a bit that this is the result of the operation.

> After you run the example, the document will be updated to the following values:

Or something more eloquent than that.

Reply:
noted, changed the phrasing